.evergreen/run-mongodb-oidc-test.sh

```diff
@@ -34,7 +34,12 @@ fi
 which java
 export OIDC_TESTS_ENABLED=true
 
-./gradlew -Dorg.mongodb.test.uri="$MONGODB_URI" \
+# use admin credentials for tests
+TO_REPLACE="mongodb://"
+REPLACEMENT="mongodb://$OIDC_ADMIN_USER:$OIDC_ADMIN_PWD@"
+ADMIN_URI=${MONGODB_URI/$TO_REPLACE/$REPLACEMENT}
```

Comment:
This looks wrong. What's the intent?

Reply:
This creates a URI that is able to run admin commands.

Reply:
sed...